src/router/Routes.tsx

```diff
@@ -71,13 +75,29 @@ export function Home() {
     );
 }
 
+export function EditProfile() {
+
+    let {userId} = useParams();
+
+    return (
+        <div>
+            <TopBar isAdmin={false}/>
+            <div className={"side-bar-container"}>
+                <SideBar isAdmin={false}/>
+                <DeleteButton pathVariable={userId}/>
```

Comment:
[Comment, no actions required] The button and the modal should be different components. The button should only render a button and have some props like onClick, label, disable, etc. 

On the other hand, the modal should only render something (ideally another component) it doesn't have to manage its own open/close functionality this should be managed by the component using the modal.